### ISSUE TITLE
failed to get pdata

### ISSUE THREAD

**Issue by selfpaidinc** [skiplagged/pokemongo-php#29]
I've managed to get this far but can't get past this error. 

getSpecificApiEndpoint() throws "failed to get pdata"

I printed the response it checks to throw that error and it's only returning;

Array ( [success] => )

**Comment by sino05** [comment]
I'm currently experiencing pretty much the same problem.
After loggin in, when getSpecificApiEndpoint() is getting called the following error is displayed:
array(1) { ["success"]=> bool(false) } 
Exception: failed to get pdata in 

I think this is a server issue or some kind of (temporary) block.

**Comment by baskettcase** [comment]
Yeah I think SkipLagged is having server issues, I edited some of my errors and this is what I am getting: 

response = Array
(
    [success] => 
)
1470147243.495 failed to get pdata from SKIPLAGGED (server not responding)

**Comment by teckel12** [comment]
Same issue here.

**Comment by Raphaelbm** [comment]
Same

**Comment by IsseiMori** [comment]
same

**Comment by wearelogik** [comment]
This project is considered dead.

**Comment by morinted** [comment]
Anecdotal, but I'm running skiplagged/pokemongo-python and it's working fine.

**Comment by wearelogik** [comment]
Not even the sligtest. With absolutely no response from the creator and a endpoint that clearly changes constantly, then it's pretty likely.

I assumed the Python API was working as he his still scanning for Pokemon on his own website.

**Comment by morinted** [comment]
It's all working. Maybe you got IP blocked at some point and it hasn't undone? 
<img width="495" alt="screen shot 2016-08-02 at 4 32 21 pm" src="https://cloud.githubusercontent.com/assets/5840970/17344052/b85a26e4-58ce-11e6-9f91-d27d0220cf86.png">

**Comment by baskettcase** [comment]
Wow so the PHP version is working for you then @morinted ? Im having the same issue where no pdata is being returned.

**Comment by Raphaelbm** [comment]
the PHP version doesn't work for me. I don't know why... I need to access to this API !

**Comment by wearelogik** [comment]
No, my IP isn't blocked! And if it is, then it's blocked completely out of proportions. Considering that i still havnt been able to get this library to work properly and it has never been used live except for testing purposes during coding.

**Comment by morinted** [comment]
Sorry, I am using a Python library @baskettcase, but this is just to confirm that the API isn't down. I had some unclear errors when my PTC login credentials were incorrect. You need to use username and password, not the email, if that helps. May be worth to try the Python version to see if it's a difference between the code.

**Comment by baskettcase** [comment]
Thanks @morinted yeah Im a bit baffled.. everything was working fine, but now it is not, so Im pretty sure all my credentials, etc are working fine. When my point data is sent to skiplagged that's when I get nothing back. Maybe he reduced the size of the bounds.. what bounds are you using? Ill try that and see if that works?

**Comment by morinted** [comment]
Hm, did you confirm the email on your PTC account? They expire after two days if you don't verify.

**Comment by baskettcase** [comment]
Yes I did and I logged in through another library and logged in directly to the account in the game, all are working just fine. When I request an access key from PTC I get it, it's when I communicate with SKIPLAGGEd that's when I get back false from them.

**Comment by morinted** [comment]
Interesting, thanks. My bounds are about 8 city blocks wide. Have you tried the Python version just to see if it's this particular repo's fault?

**Comment by wearelogik** [comment]
Should he need to? Wouldnt it be better if the "maintainer" just replied ? Instead of you, a random guy, wasting time with pure guesses?

**Comment by baskettcase** [comment]
I downloaded it, but haven't tried it. I checked the code and it's pointing to the same pokemon.php file on the skiplagged domain, so Im guessing that it is still working.. oh the developer left this thread. :(

**Comment by wearelogik** [comment]
And i repeat myself, the developer is vague and with-holding and answer questions as he seems fit in the way he seems fit.

This library is a pure waste of time.

**Comment by aktarer** [comment]
Hey guys, sorry for my lack of replies--I have been busy scaling the back end. We have literally over 30,000 people using that API endpoint per second now. Things should be a lot smoother in a few hours after we deploy some optimizations. If you're having issues, send me a direct email and we can investigate together: aktarer@skiplagged.com

**Comment by morinted** [comment]
@wearelogik okay, barring everything else, your comments have simply been unproductive. It's not random guesses, it's a process of debugging. It's up to me what I do with my time, even if I'm "wasting" it. Not replying for a day or two is not "abandoning" a project. Entitled and rude comment, like yours (though that may not have been intentional), make open source contributors burn out. We should all be trying to help each other out instead of drawing conclusions or slinging crud around. Anyways, it's been a great chat.

@aktarer clearly we've gone off track, feel free to delete the discussion if you'd like to clean up the issue.

**Comment by wearelogik** [comment]
@morinted There is no reply of days, weeks, when he does reply you get very vague comments just like the one he posted above. There is no information, there is no help, and in this case he's even running awway from the entire issue tracker on github and refer to email. He could just as well close the entire issue tracker.

The problem isn't a un-responsive server and a un-responsive maintainer for the past 24 hours. It's been the case since day one. Just read the issues, they are mostly ignored by the developer and answered by the users doing pure guess work.

Aktarer is not doing us a favor by giving us this library, we are doing him a favor by using it. That being so, i expect a lot more information to be given back and forth, not have all your questions ignored at the authors discretion.

**Comment by morinted** [comment]
Man… remember the human. If you want to move on, do so in a productive manner. Open source only works because there are good people. If you somehow believe that Skiplagged owes you for running a server and for you aren't okay with that, then just move on. If you think the maintainer isn't good, then either contribute or move on. You don't know this guy's job, how much free time he has, his skill level. I'm not saying it's one way or the other, maybe somehow he's truly got the worst intentions, but even so that's no excuse to assume and be rude about it.

There's so much opportunity to be a reasonable human being on GitHub yet you're complaining about this free project. All I came to say originally is that it's working for me, which is an indicator it's not dead. Sorry that I got you (and myself) worked up about it.

**Comment by wearelogik** [comment]
Your apology is accepted. Move along.

**Comment by baskettcase** [comment]
Hi @aktarer I sent you an email earlier today. I hope to hear from you soon!

**Comment by Raphaelbm** [comment]
I send you an email too @aktarer ! :+1:

**Comment by sino05** [comment]
I send you an email too @aktarer .

to @baskettcase & @Raphaelbm , any results so far? Are you able to retrieve data?

**Comment by Raphaelbm** [comment]
Nope, i can't retrieve data and no answer !

**Comment by wearelogik** [comment]
....

**Comment by IsseiMori** [comment]
@morinted I tried Python library, and it had been working these days but not anymore. Is your script still working?

**Comment by baskettcase** [comment]
@IsseiMori none of the libraries are working right now when it comes to the Map API. Niantic changed things in this last update to block people. Right now a lot of developers are working on cracking the code.

**Comment by IsseiMori** [comment]
@baskettcase Thank you for your reply. I understand it, however, I'm confused because Skiplagged map is still working. https://skiplagged.com/pokemon/#37.804,-122.416,15

**Comment by baskettcase** [comment]
@IsseiMori @aktarer Ill have to let the dev answer that one.. very strange.. unless it's in cache.. Im not sure.. interesting.

**Comment by IsseiMori** [comment]
@baskettcase I made sure the data, using iPhone fake GPS, and It's working accurately right now. I guess that's because Skiplagged is getting data from users, not Pokemon Server. Therefore I think the developer of this API is blocking us for some reason, not Niantic. No offence to @aktarer

**Comment by wearelogik** [comment]
When the API's went down, so did Skiplagged. But he's been stealing information / scraping it off other sites as well.

This library never recieves any data from Skiplagged, it never has and it never will. What it did, was scan the pokemon servers for new pokemons, then send all the pokemons you found back to Skiplagged. Skiplagged was simply using your tokens and your information to fetch what THEY needed. All in all, your helping Skiplagged more then they are helping you, specially with how unresponsive this **shitty** developer is.

Forget about this library and move on! It's dead. When the API's do come back up, find another library. This isn't recommended in any way.
